refactor(ast_codegen): line breaks between types in layout assertions

overlookmotel opened the pull request:
Small style nit. Add line breaks between types in generated layout assertions, to make the file easier to read.

overlookmotel commented:
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4781?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4781** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4781?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈
* **#4778** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4778?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>: 1 other dependent PR ([#4779](https://github.com/oxc-project/oxc/pull/4779) <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4779?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>)
* **#4777** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4777?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#4775** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4775?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#4774** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4774?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#4773** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4773?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @overlookmotel and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="11px" height="11px"/> <b>Graphite</b></a>
  <!-- Current dependencies on/for this PR: -->

rzvxa reviewed (approved):
I like the idea, However, I've made some comments on the `insert!` macro PR.

overlookmotel committed:
refactor(ast_codegen): line breaks between types in layout assertions (#4781)

Small style nit. Add line breaks between types in generated layout assertions, to make the file easier to read.
